packages/toolkit/src/view/pipeline/view-pipeline/PipelineRunViews/RunComponents.tsx

```diff
@@ -0,0 +1,266 @@
+            title="Total Duration"
+            paramName="total_duration"
+            currentSortParamValue={orderBy}
+            onSort={onSortOrderUpdate}
+          />
+        ),
+        cell: ({ row }) => {
+          return (
+            <div className="font-normal text-semantic-bg-secondary-alt-primary">
+              {row.getValue("totalDuration")
+                ? convertToSecondsAndMilliseconds(
+                    (row.getValue("totalDuration") as number) / 1000,
+                  )
+                : null}
+            </div>
+          );
+        },
+      },
+      {
+        accessorKey: "startTime",
+        header: () => (
+          <RunsTableSortableColHeader
+            title="Start Time"
+            paramName="started_time"
+            currentSortParamValue={orderBy}
+            onSort={onSortOrderUpdate}
+          />
+        ),
+        cell: ({ row }) => {
+          return (
+            <div className="font-normal text-semantic-bg-secondary-alt-primary">
+              {formatDateFull(row.getValue("startTime"))}
+            </div>
+          );
+        },
+      },
+      {
+        accessorKey: "completeTime",
+        header: () => (
+          <RunsTableSortableColHeader
+            title="Completed Time"
+            paramName="completed_time"
+            currentSortParamValue={orderBy}
+            onSort={onSortOrderUpdate}
+          />
+        ),
+        cell: ({ row }) => {
+          return (
+            <div className="font-normal text-semantic-bg-secondary-alt-primary">
+              {row.getValue("completeTime")
+                ? formatDateFull(row.getValue("completeTime"))
+                : null}
+            </div>
+          );
+        },
+      },
+    ];
+
+    if (env("NEXT_PUBLIC_APP_ENV") === "CLOUD") {
+      columns.push({
+        accessorKey: "creditAmount",
+        header: () => <div className="text-left">Credits</div>,
+        cell: ({ row }) => {
+          return (
+            <div className="font-normal text-semantic-bg-secondary-secondary break-all">
+              {row.getValue("creditAmount")}
+            </div>
+          );
+        },
+      });
+    }
+
+    columns.push({
+      accessorKey: "outputs",
+      header: () => <div className="text-left">Output</div>,
+      cell: ({ row }) => {
+        return (
+          <div className="font-normal text-semantic-bg-secondary-secondary break-all">
+            <Button
+              size="sm"
+              variant="secondaryGrey"
+              onClick={() => {
+                setCurrentOutputContent({
+                  componentId: row.original.componentId,
+                  content: JSON.stringify(
+                    (row.getValue("outputs") as GeneralRecord)[0],
+                    null,
+                    2,
+                  ),
+                });
+              }}
+            >
+              JSON
+            </Button>
+          </div>
+        );
+      },
+    });
```

Comment:
@orangecoloured If there has no condition for this last push, I think we can directly add it into initial array

Reply:
In the columns list the credits comes before the Output, that's why it's like this here.

Reply:
Oh I see!! That make sense